.github/workflows/add-labels.yml

```diff
@@ -0,0 +1,17 @@
+name: add-labels
```

Comment:
Is this for adding to ACE's health dashboard?

Reply:
Yes Bishal